### PR TITLE
Update README's and package.json's

### PR DESCRIPTION
**Pull request by espendk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,21 +4,39 @@ This is the Mangrove monorepo which contains most of the packages developed for
...
+
 # Prerequisites
 You must have [Yarn 2](https://yarnpkg.com/) installed, as this monorepo uses [Yarn 2 workspaces](https://yarnpkg.com/features/workspaces) to manage dependencies and run commands on multiple packages. You must also have [Node.js](https://nodejs.org/en/) 14.14+.
 
 
 # Usage
 The following sections describe the most common use cases in this monorepo. For more details on how to use Yarn and Yarn workspaces, see the [Yarn 2 CLI documentation](https://yarnpkg.com/cli/install).
 
-⚠️&nbsp; Be aware that when googling Yarn commands, it's often not clear whether the results pertain to Yarn 1 (aka 'Classic') or Yarn 2. Currently (September 2021), most examples and much tool support is implicitly engineered towards Yarn 1.
+⚠️&nbsp; Be aware that when googling Yarn commands, it's often not clear whether the results pertain to Yarn 1 (aka 'Classic') or Yarn 2. Currently (November 2021), most examples and much tool support is implicitly engineered towards Yarn 1.
```
Comment (dontrolle) [inline README.md:19]: Confusingly, the version of Yarn that we're using is strictly speaking actually Yarn 3.0.x (at least on my machine). I think the most common colloquial term to use for _new_ Yarn is either Yarn 2+ or maybe Yarn Berry (it's actual name). 

I'm a bit dissappointed by the docs for Yarn, in general, so maybe we should make sure that our users aren't also confused here.

File: packages/commonlib.js/README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+$ yarn build     # Builds the commonlib and its dependencies
+```
+
+Tests are executed with
+
+```shell
+$ yarn test
+```
+
+When changing this library, you should build and test the whole repo, to ensure that packages depending on commonlib.js still work:
```
Comment (dontrolle) [inline packages/commonlib.js/README.md:41]: (Minor quibble) "changing" -> "makes changes in"

File: packages/mangrove-solidity/README.md [hunk 3]
```diff
@@ -1,52 +1,102 @@
...
 
-- interpreting `console.log`-type statements
-- displaying Solidity stack traces
+- [./deploy/mangrove.js](./deploy/mangrove.js): Can be used for test network deployments and includes some IERC20 test tokens. This is particularly useful for setting up local (Hardhat) networks for integration tests against Mangrove.
+- [./deploy/mumbai.js](./deploy/mumbai.js): Deployment script for the Polygon Mumbai testnet.
+
+# Generate documentation
 
-It has an extendable task system; this repo adds a `test-solidity` task. Run `npx hardhat help test-solidity` for available options. The task itself is defined in [hardhat.config.js](./hardhat.config.js).
+The Mangrove Solidity files contains documentation that can be extracted to a nicely formatted and navigable HTML file by running `yarn doc` which will generate a `doc/MgvDoc.html`.
```
Comment (dontrolle) [inline packages/mangrove-solidity/README.md:98]: contains -> contain

**Review by dontrolle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (espendk) [inline README.md:19]:
```suggestion
⚠️&nbsp; Be aware that when googling Yarn commands, it's often not clear whether the results pertain to Yarn 1 (aka 'Classic') or Yarn 2+. Currently (November 2021), most examples and much tool support is implicitly engineered towards Yarn 1.
```

File: packages/commonlib.js/README.md (hunk 2 above)
Comment (dontrolle) [reply packages/commonlib.js/README.md:41]:
```suggestion
When making changes in this library, you should build and test the whole repo, to ensure that packages depending on commonlib.js still work:
```

File: packages/mangrove-solidity/README.md (hunk 3 above)
Comment (dontrolle) [reply packages/mangrove-solidity/README.md:98]:
```suggestion
The Mangrove Solidity files contain documentation that can be extracted to a nicely formatted and navigable HTML file by running `yarn doc` which will generate a `doc/MgvDoc.html`.
```

**Review by dontrolle — APPROVED** [review]

**Pull request merged**